Extension/src/LanguageServer/ui.ts

```diff
@@ -118,7 +125,9 @@ export class LanguageStatusUI {
             title: this.configureIntelliSenseStatusItem.name,
             arguments: ['statusBar']
         };
-        this.ShowConfigureIntelliSenseButton(false, this.currentClient);
+        if (this.currentClient !== undefined) {
```

Comment:
It seems like this does the opposite of what it did before.  It used to pass in the currentClient, now it doesn't even pass in a client.  Is that intentional?